src/components/Button/styledicon.js

```diff
@@ -1,6 +1,9 @@
 import styled from 'styled-components';
 
 const StyledIcon = styled.button`
```

Comment:
The name `StyledIcon` for a button is a bit misleading. When reading it first, I thought: Why is there an `onClick` event on an icon and not on a button? 
Maybe rename ist to `IconButton` or something like that, so that it is clear that it is a button and not just an icon :)